js-exercise-2/cart.js

```diff
@@ -0,0 +1,73 @@
+var getData = JSON.parse(localStorage.getItem('CART'));
+var $box = document.getElementById('js-cart-list');
+var $trTotal = document.getElementById('js-cart-total');
+var j;
+function showCart() {
+  $box.innerHTML = '';
+  for (var i = 0; i < getData.length; i++) {
+    j = i + 1;
+    $box.innerHTML +=
+      '<tr id="js-remove' + j + '" >' +
+      '<td class="td-product-stt" id="js-stt">' + j + '</td>' +
+      '<td class="td-image"><img class="td-cart-img" src="./images/' + getData[i].object.image + '"></td>' +
+      '<td class="td-product-name">' + getData[i].object.name + '</td>' +
+      '<td class="td-product-quantity">' + getData[i].quantity + '</td>' +
+      '<td class"td-product-price">$' + getData[i].object.price + '</td>' +
+      '<td class="td-product-sub-total">$' + (getData[i].object.price * getData[i].quantity) + '</td>' +
+      //'<td class="td-product-delete" data-id="1" onclick="deleteItem(' + getData[i].object.id + ')"><i class="fa fa-times"></i></td>' +
+      '<td class="td-product-delete"><button type="button" class="js-bt-delete" data-id="' + getData[i].object.id + '">X</button></td>' +
+      '</tr>';
+  }
+  $trTotal.innerHTML = '';
+  $trTotal.innerHTML +=
```

Comment:
In the `tfoot` we only show the total, so i think we don't need to create an HTML for total here
I have another solution for you.
Create an HTML like example below
```
<tfoot>
  <tr>
    <td></td>
    <td></td>
    <td></td>
    <td id="total"></td>
  </tr>
</tfoot>
```
add put it to HTML file. After that, you selector to DOM `total` and add number total for it.